PowerShell/ScubaGear/Rego/AADConfig.rego

```diff
@@ -280,6 +282,11 @@ tests contains {
 # MS.AAD.3.4v1
 #--
 
+PolicyMigrationIsComplete := Status if {
+    some Policy in input.authentication_method
+    Status := Policy.PolicyMigrationState == "migrationComplete"
+}
+
```

Comment:
because you are looping through authentication_method, isn't it possible there is multiple `PolicyMigrationState` keys. I only see unit tests for 1 "Policy", do you know how this behaves with multiple?

Reply:
I am fairly sure that there is only one - we seem to wrap all SDK results in an array for some reason.